tests/test_cosine_similarity.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+import nlu
+
+@pytest.fixture()
+def source():
+    return ['我', '喜歡', '看', '電視', '不', '喜歡', '看', '電影']
+
+
+@pytest.fixture()
+def target():
+    return ['我', '不', '喜歡', '看', '電視', '也', '不', '喜歡', '看', '電影']
```

Comment:
At here, I don't think `fixture` is better than a variable in test function.

Remember `fixture` is not an easy thing to get it for a reader that never seem it before

Reply:
Ok, I will fix it.